Sources/Charcoal/Filters/FilterViewController.swift

```diff
@@ -59,19 +63,26 @@ class FilterViewController: ScrollViewController, FilterBottomButtonViewDelegate
         super.viewWillAppear(animated)
         view.bringSubviewToFront(bottomButton)
         enableSwipeBack(true)
+        updateBottomShadow()
```

Comment:
Why do we need to do this on `viewWillAppear`?

Reply:
We have to check the content size and see if we need a shadow before scrolling.